### DISCUSSION TITLE
when using in wordpress some resize images plugin cant working , always show 413 Request Entity Too Large

### DISCUSSION THREAD

**Discussion by zavierlau** [nginx-proxy/nginx-proxy#2205] [Q&A]
Hi , if i using the proxy some images resize plugin ( smush pro ) not working, always show   "The server cannot process the image. This can happen if the server is busy or does not have enough resources to complete the task. Uploading a smaller image may help. Suggested maximum size is 2560 pixels."      

if without proxy all is good. can help on this? thanks.

**Comment by SchoNie** [comment]
You could try to increase the [`client_max_body_size`](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size)
The [Per-VIRTUAL_HOST example in readme](https://github.com/nginx-proxy/nginx-proxy#per-virtual_host) exactly shows how to do this for `client_max_body_size 100m;`
